source/Detector/Scaler/AMDSScalerDetector.cpp

```diff
@@ -229,25 +233,25 @@ void AMDSScalerDetector::onFetchScanBuffer()
 	}
 }
 
+void AMDSScalerDetector::onTriggerDwellInterfaceDwellModeControlValueChanged(double value)
+{
+	Q_UNUSED(value)
+
+	if(triggerDwellInterfaceDwellModeControl_->withinTolerance(1.0) && triggerDwellInterfaceStartControl_->withinTolerance(1.0)) {
```

Comment:
Just wanting to double check that these are both supposed to be at 1 if we are wanting to stop acquisition?